assignment-2-ruby/rborder/tiny_instacart.rb

```diff
@@ -0,0 +1,119 @@
+require './order_items_data'
+require 'time'
+
+# CUSTOMER APP
+
+# select a store
+def select_warehouse
+  selected_store = ""
+  stores = ['Kroger','Publix','Aldi','Sprouts']
+  stores.each do |store|
+    print "| #{store} |"
+  end
+  puts "\n"
+  while stores.include?(selected_store) == false do
```

Comment:
could do `while !stores.include?(selected_store) do`

Reply:
Ah! Bang operators. May take a little to get the muscle memory there.